packages/shared/lib/core/wallet/utils/getBech32AddressFromAddressTypes.ts

```diff
@@ -2,9 +2,9 @@ import { api } from '@core/profile-manager'
 import { Address, AddressType, AliasAddress, Ed25519Address, NftAddress } from '@iota/sdk/out/types'
 import { getNetworkHrp } from '@core/profile'
 
-export function getBech32AddressFromAddressTypes(address: Address): string | undefined {
+export function getBech32AddressFromAddressTypes(address: Address | undefined): string | undefined {
```

Comment:
I think the bugfix for this issue should not come from making this argument optional, as it must be compulsory that you pass an address in order to be able to retrieve its bech32, what needs fixing is all the type errors we have in the code where we are actually calling this function with something that might be undefined, eg
![image](https://github.com/iotaledger/firefly/assets/3624944/ce5b01e1-df95-4b81-963b-428ee25bb32b)
We need to make sure in the code that we are not calling this with something that is undefined, and only call it when the types are right